reference/6/Microsoft.PowerShell.Management/Set-Location.md

```diff
@@ -45,7 +45,7 @@ stack. For more information about location stacks, see the Notes.
 ### Example 1: Set the current location
 
 ```powershell
-PS C:\> Set-Location -Path "HKLM:"
+Set-Location -Path "HKLM:"
```

Comment:
Since we are documenting `Set-Location`, we should show the prompt here to illustrate how the prompt is affected by the change in location.